chapter07/kds/ex4.pl

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+print "Enter some text or supply an input file as an argument:\n";
+while( <> ){
+  print "$_" if( /[A-Z][a-z]/ );
```

Comment:
Don't think this would match MMMBop (the catchy 1990's Hanson hit pop lovers :musical_note: ) - capitalized but not ALL capitalized